src/traces/cone/attributes.js

```diff
@@ -96,13 +96,14 @@ var attrs = {
 
     sizemode: {
         valType: 'enumerated',
-        values: ['scaled', 'absolute'],
+        values: ['scaled', 'absolute', 'vector'],
```

Comment:
Another option instead of calling it *vector* is *uvw* which refers to the data arrays the size is coming from.